.buildkite/pipelines/on_merge.yml

```diff
@@ -103,7 +113,12 @@ steps:
         - exit_status: '*'
           limit: 1
 
-  - command: .buildkite/scripts/steps/functional/security_serverless_investigations.sh
+  - command: .buildkite/scripts/steps/functional/security_cypress_exec.sh
+    env:
+      TARGET: cypress:investigations:run:serverless
+      CURRENT_WORKING_DIR: x-pack/test/security_solution_cypress
+      JOB_TITLE: kibana-security-solution-chrome
```

Comment:
Do we want JOB_TITLE to be the same for multiple jobs?  Since they run different targets it seems like the job title should reflect that and be repeated.

Reply:
I copied exactly what they were using so far. I dont know what functionality lives behind this env var